web-registry/src/pages/ProjectFinished/ProjectFinished.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Box } from '@mui/system';
+
+import OnBoardingCard from 'web-components/lib/components/cards/OnBoardingCard';
+import { Title } from 'web-components/lib/components/typography';
+import { CardItem } from 'web-components/lib/components/modal/TxModal';
+import OutlinedButton from 'web-components/lib/components/buttons/OutlinedButton';
+import { truncateHash } from 'web-components/lib/utils/truncate';
+
+import { OnboardingFormTemplate } from '../../components/templates';
+import { Link } from '../../components/atoms';
+import { getHashUrl } from '../../lib/block-explorer';
+import { useCreateProjectContext } from '../../features/ecocredit/CreateProject/CreateProjectContext';
+import { useGetProjectId } from './hooks/useGetProjectId';
+
+const ProjectFinished: React.FC = () => {
+  const { deliverTxResponse } = useCreateProjectContext();
+  const projectId = useGetProjectId(deliverTxResponse);
+
+  return (
+    <OnboardingFormTemplate activeStep={2} title="Project has been created!">
+      <Box
+        sx={{
+          display: 'flex',
+          flexDirection: 'column',
+          justifyContent: 'center',
+        }}
+      >
+        <OnBoardingCard>
+          <Title variant="h5">Create Project</Title>
+          <CardItem
+            label="project id"
+            value={{
+              name: projectId,
+            }}
+            linkComponent={Link}
+          />
+          <CardItem
+            label="hash"
+            value={{
+              name: truncateHash(deliverTxResponse?.transactionHash) || '',
+              url: getHashUrl(deliverTxResponse?.transactionHash),
+            }}
+            linkComponent={Link}
+          />
+        </OnBoardingCard>
+        <OutlinedButton
+          sx={{ margin: '0 auto' }}
+          href={`/projects/${projectId}`}
+        >
```

Comment:
we'll wanna `useNavigate` and change this to something like
```suggestion
        <OutlinedButton
          sx={{ margin: '0 auto' }}
          role="link"
          onClick={() => navigate(`/projects/${projectId}`)}
        >
```
or it'll trigger a page refresh. Not the most accessible, and there are MUI ways of handling this where a button can be rendered as a React Router `Link` and function appropriately, but they don't work in our current system right now (would have to build separate components I think, or handle through theme) so `navigate` is probably the easiest way for now

Reply:
thx